src/main/java/org/sonarlint/intellij/actions/OpenSecurityHotspotInDialogAction.kt

```diff
@@ -0,0 +1,19 @@
+package org.sonarlint.intellij.actions
+
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.Project
+
+class OpenSecurityHotspotInDialogAction : AbstractSonarAction(
```

Comment:
I would suggest to use a more precise name, what is really the purpose of the action ? Something like `OpenHotspotStatusChangeDialogAction`